pom.xml

```diff
@@ -450,6 +450,12 @@
                         <exclude>tools/dependencies/known-oap-backend-dependencies.txt</exclude>
                         <exclude>apm-checkstyle/CHECKSTYLE_HEAD</exclude>
 
+                        <!-- generated file from oap or webapp logs files -->
+                        <exclude>**/apm-dist/logs/**</exclude>
```

Comment:
Why are logs here?

Reply:
If there is an error message during operation, it will be recorded here.
![image](https://user-images.githubusercontent.com/2892433/72219747-4652ad00-3584-11ea-9c6a-df8b0202e61f.png)

Reply:
I am saying why these logs exist. As nothing should be run there.

Reply:
<img width="900" alt="WX20200113-114517@2x" src="https://user-images.githubusercontent.com/2892433/72232137-2cec4800-35fa-11ea-8503-91999a8c7224.png">

Reply:
But it should not start there. That is my point.

Reply:
I ca n’t find it now. I looked at the time when the log was generated. It ’s been a long time ago. Indeed, from the current point of view, there is no entry to generate a log.
<img width="768" alt="WX20200129-095628@2x" src="https://user-images.githubusercontent.com/2892433/73321394-4db0d080-427d-11ea-89f5-69872a86f6fc.png">